src/styles/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  margin: 50px;
+  display: flex;
+  justify-content: center;
+  width: 198px;
+  height: 406px;
```

Comment:
don't set height of element directly, use elements inside and their paddings, margins